test/policy-collection/policy_iam_test.go

```diff
@@ -0,0 +1,99 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/open-cluster-management/governance-policy-framework/test/common"
+	policiesv1 "github.com/open-cluster-management/governance-policy-propagator/pkg/apis/policy/v1"
+	"github.com/open-cluster-management/governance-policy-propagator/test/utils"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const iamPolicyName = "policy-limitclusteradmin"
+const iamPolicyURL = "https://raw.githubusercontent.com/open-cluster-management/policy-collection/main/stable/AC-Access-Control/" + iamPolicyName + ".yaml"
+
+func getComplianceState() interface{} {
```

Comment:
Can we make this take the name of the policy and namespace, and bring it into the common package? I'd be surprised if other tests can't use something like this.

Reply:
Or I guess it could go into propagator/test/utils, but I personally don't like that as a dependency. I'm never sure (without testing) how that dependency will get updated.

Reply:
Good idea. I'll make this a separate commit.

Reply:
@JustinKuli is the first commit what you had in mind?